File: Pavolle.BES.AuthServer.ViewModels/Response/SignInResponse.cs
using Pavolle.BES.AuthServer.ViewModels.ViewData;
using Pavolle.Core.ViewModels.Response;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Pavolle.BES.AuthServer.ViewModels.Response
{
    public class SignInResponse : ResponseBase
    {
        public string Token { get; set; }
        public List<UserAuthorizationViewData> Authorizations { get; set; }
        public SingInUserDetailViewData UserDetail { get; set; }
    }
}

File: Pavolle.BES.PasswordServer.WebApp/ClientApp/src/app/viewmodels/response/sign-in-response.ts
import { SingInUserDetailViewData } from "../viewdata/sign-in-user-detail-view-data";
import { UserAuthorizationViewData } from "../viewdata/user-authorization-view-data";
import { ResponseBase } from "./response-base";

export interface SignInResponse extends ResponseBase {
    token: string;
    authorizations: UserAuthorizationViewData[];
    userDetail: SingInUserDetailViewData;
}
